lib/logger.js

```diff
@@ -1,16 +1,39 @@
 import npmlog from 'npmlog';
 
-const npmLevels = ['silly', 'verbose', 'info', 'http', 'warn', 'error'];
+const npmLevels = ['silly', 'verbose', 'debug', 'info', 'http', 'warn', 'error'];
 let mockLog = {};
 for (let l of npmLevels) {
   mockLog[l] = () => {};
 }
 
-export function getLogger () {
+export function patchLogger(logger) {
+  if (!logger.debug) {
+    logger.addLevel('debug', 1000, { fg: 'blue', bg: 'black' }, 'dbg');
+  }
+}
+
+function _getLogger () {
   const testingMode = parseInt(process.env._TESTING, 10) === 1;
   const forceLogMode = parseInt(process.env._FORCE_LOGS, 10) === 1;
-  return (testingMode && !forceLogMode) ? mockLog :
-                                          (global._global_npmlog || npmlog);
+  let logger = (testingMode && !forceLogMode) ? mockLog :
+    (global._global_npmlog || npmlog);
+  patchLogger(logger);
+  return logger;
+}
+
+export function getLogger(prefix=null) {
```

Comment:
we need to standardize on either `(prefix=null)` or `(prefix = null)`. I've been doing the latter